lib/auth/legacy.js

```diff
@@ -52,7 +52,7 @@ function login (conf) {
     })
     .catch((err) => {
       if (err.code !== 'EOTP') throw err
-      return read.otp('Authenticator provided OTP:').then((otp) => {
+      return read.otp().then((otp) => {
```

Comment:
In retrospect: I don't think we should change this one. `npm login` is intended to be a pretty interactive workflow and I don't think it's useful to tell people this is an error or dump a bunch of text on them. I think we can leave this one as-is.